platform/android/os_android.cpp

```diff
@@ -423,6 +423,15 @@ String OS_Android::get_clipboard() const {
 	return OS_Unix::get_clipboard();
 }
 
+bool OS_Android::has_clipboard() const {
+	// DO we really need the fallback to OS_Unix here?!
```

Comment:
I don't think we do, but given that's the existing pattern, let's go with that for now and I can clean up this logic later.